app/src/main/res/values/strings.xml

```diff
@@ -26,154 +26,154 @@
-    <string name="msg_server_profile_deleted">Server deleted.</string>
-    <string name="msg_pip_not_supported">Picture-in-Picture is not supported on this device</string>
-    <string name="msg_ptr_hiding_not_supported">Hiding cursor is not supported on this device</string>
-    <string name="msg_loading">Loading …</string>
-    <string name="msg_imported">Imported Successfully</string>
-    <string name="msg_exported">Exported Successfully</string>
-    <string name="msg_export_auth_required">Authentication is required before Export.</string>
-    <string name="msg_export_auth_missing">Your device is not secured. Turn on screen lock to enable authentication before Export.</string>
-    <string name="msg_invalid_key_file">File is not a valid private key!</string>
+    <string name="msg_about_app">A copylefted libre VNC client.\nUse any device with a VNC server remotely.</string>
+    <string name="msg_server_profile_added">Added</string>
+    <string name="msg_server_profile_deleted">Deleted</string>
+    <string name="msg_pip_not_supported">Not supported on this device</string>
+    <string name="msg_ptr_hiding_not_supported">Not supported on this device</string>
+    <string name="msg_loading">Loading…</string>
+    <string name="msg_imported">Imported</string>
+    <string name="msg_exported">Exported</string>
+    <string name="msg_export_auth_required">Authenticate before exporting.</string>
+    <string name="msg_export_auth_missing">Turn on screen lock to require authenticating to export servers and passwords.</string>
+    <string name="msg_invalid_key_file">This file is not a valid private key.</string>
     <string name="msg_invalid_vnc_uri">Invalid VNC URI</string>
-    <string name="msg_copied_to_clipboard">Copied!</string>
-    <string name="msg_key_compat_mode_help">Some servers don\'t support direct Unicode input. Compatibility Mode tries to send key events in legacy format.\n\nTry enabling/disabling this if you are having trouble entering some characters.</string>
-    <string name="tip_empty_server_list">Server list is empty.\nUse the \'<b>+</b>\' button to add a Server, or\nUse the top Urlbar to directly connect to a server.</string>
-    <string name="tip_urlbar">You can enter hostname or IP address.\n\'vnc://\' URIs are also supported.</string>
-    <string name="tip_discovery">Discovery can automatically find supported servers\nrunning on your network.</string>
+    <string name="msg_copied_to_clipboard">Copied</string>
+    <string name="msg_key_compat_mode_help">Compatiblity mode can help input characters on servers that don\'t support Unicode.</string>
+    <string name="tip_empty_server_list">Click \'<b>+</b>\' to add a server, or\nuse the top bar to connect to one.</string>
+    <string name="tip_urlbar">Enter a hostname or IP address.\n\'vnc://\' URIs are also supported.</string>
+    <string name="tip_discovery">Discovery can find servers on your network.</string>
```

Comment:
```suggestion
    <string name="tip_discovery">Discovery can automatically find servers on your network.</string>
```
What do you think? Having `automatically` emphasizes the automatic nature.

Reply:
As opposed to how?